site/docs/components/navigation-item/index.mdx

```diff
@@ -0,0 +1,14 @@
+---
+title: Navigation Item 🚧
```

Comment:
The icon should be on the sidebar label only otherwise it will also be displayed in the page title :)

Reply:
e.g.
```
sidebar:
  label: Navigation Item 🚧
```